apps/typescriptcourse/src/components/navigation.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import DarkModeToggle from './color-mode-toggle'
+import Link from 'next/link'
+import {useViewer} from '@skillrecordings/viewer'
+
+export type NavigationProps = {title?: string}
+
+const Navigation: React.FC<NavigationProps> = ({title = 'Product'}) => {
+  const {isAuthenticated, logout} = useViewer()
```

Comment:
gneerally I doubt if ANY context/provider would work if it's in another package like this